AsyncDisplayKit/Layout/ASDimension.mm

```diff
@@ -38,7 +38,7 @@ ASOVERLOADABLE ASDimension ASDimensionMake(NSString *dimension)
     // Check if percent is at the end and remove it
```

Comment:
@maicki above, in the dimension.length == 0 case, should we maybe return ASDimensionAuto and assert?